server/controllers/officeController.js

```diff
@@ -1,91 +1,142 @@
-import DataModel from '../models/DataModel';
-const officeFileJson = '../helper/data/offices.json';
-const offices = require(officeFileJson);
+import Runner from '../../config/Runner';
+
+const queryInsert = 'INSERT INTO offices(name, type) VALUES ($1, $2) returning *';
+const queryEdit = 'UPDATE offices SET name=$1, type=$2, updated_at=NOW() WHERE id=$3 returning *';
+const queryAll = 'SELECT * FROM offices';
+const queryOne = 'SELECT * FROM offices WHERE id = $1';
+const queryDelete = 'DELETE FROM offices WHERE id = $1';
 
 const officeController = {
 	createNewOffice(req, res){
-		req.body.type = req.body.type.toLowerCase().trim();
-		DataModel.saveNew(officeFileJson, offices, req.body)
-			.then(office => {
-				res.status(201).json({
-					status: 201,
-					message: 'Successfully created',
-					data: office
+		const values = [
```

Comment:
@AJAkimana This PR's description is `Create signup`, so it's scope is limited to things `signup`. Why do we have code for other functionalities?